docker/build/Training.ipynb

```diff
@@ -69,19 +67,12 @@
    "source": [
     "for i, random_seed in enumerate([14, 18, 20, 30]):\n",
     "    print(\"\\n\\n********* STARTING TRAINING FOR RANDOM SEED {} ({}/4) *********\\n\\n\".format(random_seed, i))\n",
-    "    train_config = {\"session\": i, \"max_epochs\": 14, \"num_workers\": 2, \"optimizer\": \"sgd\", \"batch_size\": 4, \n",
+    "    train_config = {\"session\": i, \"max_epochs\": 14, \"optimizer\": \"sgd\", \"batch_size\": 4, \n",
```

Comment:
why did you take off `num_workers`?

Reply:
because the default behavior is to use 4

Reply:
is it not necessary?

Reply:
please reintroduce it, if they want to change it to some other number depending on the number of gpus they use

Reply:
it's not necessary tho to re-produce the results. we already have a disclaimer that if they want to re-produce our results they should use 4 GPUs.

Reply:
you will reintroduce it or not?

Reply:
Can’t hurt to include it here, it should be set to 4. Whatever the default is.